### ISSUE TITLE
[COGNITION]: Consider making Facility Operating Statuses a description list

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#6028]
## Description

The Operating Statuses page has a section that feels like a description list semantically, but is just marked up using general HTML. Making this a list would go a long way to helping users understand the relationship between facilities and hours. Screenshot below.

## Type of Issue
content
_Per @davidconlon, this should be handled from the front end_

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Acceptance Criteria

**Definition of done**
* [ ] Screen reader users can parse the facility operating statuses section as a list

## Environment

* MacOS Mojave
* Safari
* VoiceOver
* https://www.va.gov/pittsburgh-health-care/operating-status/

## WCAG or Vendor Guidance (optional)

* [Info and Relationships: Understanding SC 1.3.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html)

## Screenshots or Trace Logs

<img width="844" alt="Screen Shot 2020-02-19 at 5 02 04 PM" src="https://user-images.githubusercontent.com/934879/74884920-ef39b800-5339-11ea-84e1-aa036e3822df.png">

**Comment by ethanteague** [comment]
PR: https://github.com/department-of-veterans-affairs/vets-website/pull/12354

**Comment by mmiddaugh** [comment]
reopening for @jenstrickland's validation

**Comment by jenstrickland** [comment]
There appears to be a bug in Firefox with NVDA (on Windows). It reads this list as 14 items, instead of 7 like the other browser + screen reader combos.

**Comment by 1Copenut** [comment]
@jenstrickland I'm guessing Firefox + NVDA is reading it as 14 items because there's a `<div>` wrapping each `<dt><dd>` pair: https://github.com/department-of-veterans-affairs/vets-website/blob/3cc3dcdc473508bccfe835de2c833cb8f52dfa42/src/site/layouts/vamc_operating_status_and_alerts.drupal.liquid#L84

**Comment by ethanteague** [comment]
@jenstrickland @1Copenut this appears to be a known issue, unique to NVDA: https://github.com/nvaccess/nvda/issues/3858
I could change this markup to a ul/li type of structure to address, but the consensus (at least in the NVDA repo) seems to be that it's better to use DL structure, in spite of this issue.

**Comment by 1Copenut** [comment]
Thank you @ethanteague for that cross-link. Even though it's not a great experience, I'd opt to keep the structure you have, as a description list. Do you concur @jenstrickland ?

**Comment by ethanteague** [comment]
Thanks, @1Copenut - happy to change if you all want me to (it's low-effort), but yeah, dl is better.

**Comment by jenstrickland** [comment]
Since it is a known issue, and nvaccess really needs to fix, I'd say leave it. 

I tried an idea of adding aria-label="list with 7 items" on the dl, and aria-hidden="true" on each div — until I realized that was hiding everything inside the div. doh!

**Comment by 1Copenut** [comment]
I took a look at the NVDA codebase briefly, just to see if I could find where these discrepancies were hiding. The following look promising:
* https://github.com/nvaccess/nvda/blob/a335297e39a16c171a65b35f1d99fe8629e8e272/source/NVDAObjects/IAccessible/MSHTML.py#L129
* https://github.com/nvaccess/nvda/blob/16f0b903ca3f53a481b3c17e8464b6b19af957e6/source/virtualBuffers/MSHTML.py#L289

I'm no Python dev, but may fork the repo and noodle around in the off hours. Just to see if we can make an improvement.

**Comment by mmiddaugh** [comment]
@jenstrickland @1Copenut can this issue be closed?

**Comment by 1Copenut** [comment]
@mmiddaugh I believe so. It's a known issue for NVDA, so not a code issue.

@jenstrickland do you concur?

**Comment by jenstrickland** [comment]
I do indeed. Now, nvaccess… ;)
